src/test/kotlin/no/nav/syfo/identhendelse/IdenthendelseServiceSpek.kt

```diff
@@ -0,0 +1,147 @@
+package no.nav.syfo.identhendelse
+
+import io.ktor.server.testing.*
+import kotlinx.coroutines.runBlocking
+import no.nav.syfo.aktivitetskrav.database.getAktivitetskrav
+import no.nav.syfo.aktivitetskrav.domain.Aktivitetskrav
+import no.nav.syfo.client.azuread.AzureAdClient
+import no.nav.syfo.client.pdl.PdlClient
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.testhelper.generator.generateKafkaIdenthendelseDTO
+import org.amshove.kluent.internal.assertFailsWith
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.lang.IllegalStateException
+import java.time.LocalDate
+
+private val aktivIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT
+private val aktivIdentIkkeOppdatert = UserConstants.THIRD_ARBEIDSTAKER_PERSONIDENT
+private val inaktivIdent = UserConstants.OTHER_ARBEIDSTAKER_PERSONIDENT
+
+class IdenthendelseServiceSpek : Spek({
+    describe("${IdenthendelseService::class.java.simpleName}: handle") {
+        with(TestApplicationEngine()) {
+            start()
+
+            val externalMockEnvironment = ExternalMockEnvironment.instance
+            val database = externalMockEnvironment.database
+            val azureAdClient = AzureAdClient(
+                azureEnvironment = externalMockEnvironment.environment.azure
+            )
+            val pdlClient = PdlClient(
+                azureAdClient = azureAdClient,
+                pdlEnvironment = externalMockEnvironment.environment.clients.pdl,
+            )
+
+            val identhendelseService = IdenthendelseService(
+                database = database,
+                pdlClient = pdlClient,
+            )
+
+            afterEachTest {
+                database.dropData()
+            }
+
+            describe("Aktivitetskrav eksisterer for person") {
+                val aktivitetskravNy = Aktivitetskrav.ny(inaktivIdent, LocalDate.now().minusDays(50))
+                val aktivitetskravAutomatiskOppfylt =
+                    Aktivitetskrav.automatiskOppfyltGradert(inaktivIdent, LocalDate.now().minusDays(400))
+                beforeEachTest {
+                    database.createAktivitetskrav(aktivitetskravNy, aktivitetskravAutomatiskOppfylt)
+                }
+
+                it("Oppdaterer aktivitetskrav når person har fått ny ident") {
+                    var aktivitetskravMedInaktivIdent = database.getAktivitetskrav(personIdent = inaktivIdent)
+                    aktivitetskravMedInaktivIdent.size shouldBeEqualTo 2
```

Comment:
Er dette med i alle testene bare for å sjekke at `beforeEach` gjør som den skal? Det er vel i så fall ikke helt nødvendig? 🤔

Reply:
Var nok mest for å tydeliggjøre før- og etter-tilstand i testene, men enig i at de godt kan fjernes 👍